stylus/ui-app/notifications.styl

```diff
@@ -0,0 +1,54 @@
+/*!
+ * NOTIFICATIONS
+ * =============
+ *
+ * This file contains all needs for notifications
+ */
+
+
+$notifications
+    notification-width = (546/basefont)em
+    .coz-notifier
+        z-index 100
```

Comment:
The notification should on the top of all elements in the page, so I think it's why we should define here to use z-index 100 by default. What do you think?